### PR TITLE
Better usage

### PR DESCRIPTION
**Pull request by JedMeister**
Some tidying up and doc updates.

### CODE REVIEW

**Comment by JedMeister** [comment]
@OnGle

File: confconsole.py [hunk 1]
```diff
@@ -160,11 +160,8 @@ def _is_available(self):
...
-        with open('/proc/cmdline') as fh:
-            cmdline = fh.readline()
-
-        for cmd in cmdline.split():
-            if cmd == "boot=casper" or cmd == "boot=live":
+        with open('/proc/cmdline') as fob:
+            if "boot=live" in fob.readline().split():
                 return True
 
         return False
```
Comment (OnGle) [inline confconsole.py:167]: Could be further condensed to

```python
with open('/proc/cmdline') as fob:
    return 'boot=live' in fob.readline().split()
```

**Review by OnGle — APPROVED** [review]
Lgtm

File: confconsole.py (hunk 1 above)
Comment (JedMeister) [reply confconsole.py]: I did it...

**Pull request merged**